include/types.h

```diff
@@ -8,6 +8,6 @@
 #define UNK_FUN_PTR(name) void(*name)(void)
 #define UNK_ARGS
 
-typedef s32* bytecode;
+typedef s32 bytecode;
 
 #endif
```

Comment:
I don't think it's your fault, but while you're at it could you add a newline here?

Reply:
Will resolve these newline issues, I like 'em at the end of files too. This hopefully won't be an issue for most in the future thanks to #19 :package: